src/test/run-d4n-unit-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
@samarahu the tests run here are meant to be run with a vstart cluster. Could you add some documentation at the top of this file saying what's needed to setup before you run the test? Also is there anything in these tests that is not already tested in the workunit? Since these tests aren't being run continuously (ex: in make check or teuthology) are they necessary or are they just nice to have for when you're doing local development?